CHANGELOG.md

```diff
@@ -9,7 +9,64 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [`x.y.z`] - Unreleased
 
+## [`0.14.0`] - 2021-08-16
```

Comment:
This needs to be reworked from 0.14.0-rc to 0.14.0

Reply:
Yeah saw this myself, release tool just making a mistake on the last button press. Will sort and thanks for spotting.